scripts/test/test_launch_controller.py

```diff
@@ -67,10 +63,18 @@ def main():
         if args.cluster:
             extra_args += ["--cluster", "true", "--dumpAgencyOnError", "true"]
         extra_args += ["--testBuckets", args.testBuckets]
-        runner = TestingRunner(SiteConfig(Path(args.definitions).resolve()))
-        runner.scenarios.append(
-            TestConfig(runner.cfg, name, suite, [*extra_args], 1, 1, [])
-        )
+        config = SiteConfig(Path(args.definitions).resolve())
+        config.serialize_execution = True
```

Comment:
I think for better portability this flag should be set from the CLI as parameter.